system_tests/vision.py

```diff
@@ -108,3 +109,104 @@ def test_detect_logos_gcs(self):
         self.assertEqual(len(logos), 1)
         logo = logos[0]
         self._assert_logo(logo)
+
+
+class TestVisionClientFace(unittest.TestCase):
+    def setUp(self):
+        self.to_delete_by_case = []
+
+    def tearDown(self):
+        for value in self.to_delete_by_case:
+            value.delete()
+
+    def _assert_coordinate(self, coordinate):
+        if coordinate is None:
+            return True
```

Comment:
So invert the test (`if coordinate is not None:`) and make the assertions inside its suite.  Nobody is checking the return value, which makes it pointless / distracting.